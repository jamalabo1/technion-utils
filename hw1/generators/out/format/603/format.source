A X 568 1
D X 903 2
C W 206 3
B W 569 4
C Z 602 5
C X 958 6
C W 486 7
B Y 904 8
D Y 27 9
D Z 374 10
A W 845 11
D Y 148 12
D Y 30 13
D X 678 14
B W 406 15
C W 22 16
B Z 486 17
B Y 313 18
C Y 435 19
B Z 620 20
C Z 6 21
B Z 1 22
A Y 405 23
D Z 987 24
A Z 840 25
D Y 952 26
C X 490 27
B W 922 28
D W 812 29
C Z 694 30
C Y 290 31
C W 1000 32
A Y 259 33
D W 257 34
D W 728 35
A Z 782 36
A X 260 37
A X 832 38
D Y 940 39
A Z 505 40
D X 208 41
A Y 260 42
A Z 158 43
C Y 997 44
B W 472 45
A Y 186 46
C W 116 47
B W 351 48
A X 228 49
C W 689 50
C Z 356 51
B Y 402 52
C W 293 53
D W 167 54
A W 906 55
B Y 873 56
B W 148 57
C W 122 58
C Z 381 59
D Y 384 60
B W 79 61
D W 784 62
B W 918 63
A X 872 64
A W 374 65
D W 128 66
A W 713 67
C Z 201 68
D Z 349 69
C Z 889 70
D Y 44 71
C Y 467 72
B Y 87 73
D Z 461 74
C Z 341 75
A X 747 76
A Y 778 77
B Z 873 78
C Z 632 79
B W 226 80
C X 452 81
D X 327 82
D Y 100 83