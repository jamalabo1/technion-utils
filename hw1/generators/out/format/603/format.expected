Blockchain Info:
1
Sender Name: D
Receiver Name: Y
Transaction Value: 100
Transaction timestamp: 83
2
Sender Name: D
Receiver Name: X
Transaction Value: 327
Transaction timestamp: 82
3
Sender Name: C
Receiver Name: X
Transaction Value: 452
Transaction timestamp: 81
4
Sender Name: B
Receiver Name: W
Transaction Value: 226
Transaction timestamp: 80
5
Sender Name: C
Receiver Name: Z
Transaction Value: 632
Transaction timestamp: 79
6
Sender Name: B
Receiver Name: Z
Transaction Value: 873
Transaction timestamp: 78
7
Sender Name: A
Receiver Name: Y
Transaction Value: 778
Transaction timestamp: 77
8
Sender Name: A
Receiver Name: X
Transaction Value: 747
Transaction timestamp: 76
9
Sender Name: C
Receiver Name: Z
Transaction Value: 341
Transaction timestamp: 75
10
Sender Name: D
Receiver Name: Z
Transaction Value: 461
Transaction timestamp: 74
11
Sender Name: B
Receiver Name: Y
Transaction Value: 87
Transaction timestamp: 73
12
Sender Name: C
Receiver Name: Y
Transaction Value: 467
Transaction timestamp: 72
13
Sender Name: D
Receiver Name: Y
Transaction Value: 44
Transaction timestamp: 71
14
Sender Name: C
Receiver Name: Z
Transaction Value: 889
Transaction timestamp: 70
15
Sender Name: D
Receiver Name: Z
Transaction Value: 349
Transaction timestamp: 69
16
Sender Name: C
Receiver Name: Z
Transaction Value: 201
Transaction timestamp: 68
17
Sender Name: A
Receiver Name: W
Transaction Value: 713
Transaction timestamp: 67
18
Sender Name: D
Receiver Name: W
Transaction Value: 128
Transaction timestamp: 66
19
Sender Name: A
Receiver Name: W
Transaction Value: 374
Transaction timestamp: 65
20
Sender Name: A
Receiver Name: X
Transaction Value: 872
Transaction timestamp: 64
21
Sender Name: B
Receiver Name: W
Transaction Value: 918
Transaction timestamp: 63
22
Sender Name: D
Receiver Name: W
Transaction Value: 784
Transaction timestamp: 62
23
Sender Name: B
Receiver Name: W
Transaction Value: 79
Transaction timestamp: 61
24
Sender Name: D
Receiver Name: Y
Transaction Value: 384
Transaction timestamp: 60
25
Sender Name: C
Receiver Name: Z
Transaction Value: 381
Transaction timestamp: 59
26
Sender Name: C
Receiver Name: W
Transaction Value: 122
Transaction timestamp: 58
27
Sender Name: B
Receiver Name: W
Transaction Value: 148
Transaction timestamp: 57
28
Sender Name: B
Receiver Name: Y
Transaction Value: 873
Transaction timestamp: 56
29
Sender Name: A
Receiver Name: W
Transaction Value: 906
Transaction timestamp: 55
30
Sender Name: D
Receiver Name: W
Transaction Value: 167
Transaction timestamp: 54
31
Sender Name: C
Receiver Name: W
Transaction Value: 293
Transaction timestamp: 53
32
Sender Name: B
Receiver Name: Y
Transaction Value: 402
Transaction timestamp: 52
33
Sender Name: C
Receiver Name: Z
Transaction Value: 356
Transaction timestamp: 51
34
Sender Name: C
Receiver Name: W
Transaction Value: 689
Transaction timestamp: 50
35
Sender Name: A
Receiver Name: X
Transaction Value: 228
Transaction timestamp: 49
36
Sender Name: B
Receiver Name: W
Transaction Value: 351
Transaction timestamp: 48
37
Sender Name: C
Receiver Name: W
Transaction Value: 116
Transaction timestamp: 47
38
Sender Name: A
Receiver Name: Y
Transaction Value: 186
Transaction timestamp: 46
39
Sender Name: B
Receiver Name: W
Transaction Value: 472
Transaction timestamp: 45
40
Sender Name: C
Receiver Name: Y
Transaction Value: 997
Transaction timestamp: 44
41
Sender Name: A
Receiver Name: Z
Transaction Value: 158
Transaction timestamp: 43
42
Sender Name: A
Receiver Name: Y
Transaction Value: 260
Transaction timestamp: 42
43
Sender Name: D
Receiver Name: X
Transaction Value: 208
Transaction timestamp: 41
44
Sender Name: A
Receiver Name: Z
Transaction Value: 505
Transaction timestamp: 40
45
Sender Name: D
Receiver Name: Y
Transaction Value: 940
Transaction timestamp: 39
46
Sender Name: A
Receiver Name: X
Transaction Value: 832
Transaction timestamp: 38
47
Sender Name: A
Receiver Name: X
Transaction Value: 260
Transaction timestamp: 37
48
Sender Name: A
Receiver Name: Z
Transaction Value: 782
Transaction timestamp: 36
49
Sender Name: D
Receiver Name: W
Transaction Value: 728
Transaction timestamp: 35
50
Sender Name: D
Receiver Name: W
Transaction Value: 257
Transaction timestamp: 34
51
Sender Name: A
Receiver Name: Y
Transaction Value: 259
Transaction timestamp: 33
52
Sender Name: C
Receiver Name: W
Transaction Value: 1000
Transaction timestamp: 32
53
Sender Name: C
Receiver Name: Y
Transaction Value: 290
Transaction timestamp: 31
54
Sender Name: C
Receiver Name: Z
Transaction Value: 694
Transaction timestamp: 30
55
Sender Name: D
Receiver Name: W
Transaction Value: 812
Transaction timestamp: 29
56
Sender Name: B
Receiver Name: W
Transaction Value: 922
Transaction timestamp: 28
57
Sender Name: C
Receiver Name: X
Transaction Value: 490
Transaction timestamp: 27
58
Sender Name: D
Receiver Name: Y
Transaction Value: 952
Transaction timestamp: 26
59
Sender Name: A
Receiver Name: Z
Transaction Value: 840
Transaction timestamp: 25
60
Sender Name: D
Receiver Name: Z
Transaction Value: 987
Transaction timestamp: 24
61
Sender Name: A
Receiver Name: Y
Transaction Value: 405
Transaction timestamp: 23
62
Sender Name: B
Receiver Name: Z
Transaction Value: 1
Transaction timestamp: 22
63
Sender Name: C
Receiver Name: Z
Transaction Value: 6
Transaction timestamp: 21
64
Sender Name: B
Receiver Name: Z
Transaction Value: 620
Transaction timestamp: 20
65
Sender Name: C
Receiver Name: Y
Transaction Value: 435
Transaction timestamp: 19
66
Sender Name: B
Receiver Name: Y
Transaction Value: 313
Transaction timestamp: 18
67
Sender Name: B
Receiver Name: Z
Transaction Value: 486
Transaction timestamp: 17
68
Sender Name: C
Receiver Name: W
Transaction Value: 22
Transaction timestamp: 16
69
Sender Name: B
Receiver Name: W
Transaction Value: 406
Transaction timestamp: 15
70
Sender Name: D
Receiver Name: X
Transaction Value: 678
Transaction timestamp: 14
71
Sender Name: D
Receiver Name: Y
Transaction Value: 30
Transaction timestamp: 13
72
Sender Name: D
Receiver Name: Y
Transaction Value: 148
Transaction timestamp: 12
73
Sender Name: A
Receiver Name: W
Transaction Value: 845
Transaction timestamp: 11
74
Sender Name: D
Receiver Name: Z
Transaction Value: 374
Transaction timestamp: 10
75
Sender Name: D
Receiver Name: Y
Transaction Value: 27
Transaction timestamp: 9
76
Sender Name: B
Receiver Name: Y
Transaction Value: 904
Transaction timestamp: 8
77
Sender Name: C
Receiver Name: W
Transaction Value: 486
Transaction timestamp: 7
78
Sender Name: C
Receiver Name: X
Transaction Value: 958
Transaction timestamp: 6
79
Sender Name: C
Receiver Name: Z
Transaction Value: 602
Transaction timestamp: 5
80
Sender Name: B
Receiver Name: W
Transaction Value: 569
Transaction timestamp: 4
81
Sender Name: C
Receiver Name: W
Transaction Value: 206
Transaction timestamp: 3
82
Sender Name: D
Receiver Name: X
Transaction Value: 903
Transaction timestamp: 2
83
Sender Name: A
Receiver Name: X
Transaction Value: 568
Transaction timestamp: 1